bin/resources/GameIndex.yaml

```diff
@@ -12974,9 +12969,14 @@ SLES-52917:
   name: "Scaler"
   region: "PAL-E-F"
   compat: 5
+  roundModes:
+    eeRoundMode: 2  # Fixes missing text.
```

Comment:
Double space before #

Reply:
Sorry, didn't realize spacing was different now. Should be fixed.